### PR TITLE
How to create a custom create-react-app script

### PR DESCRIPTION
**Pull request by unicodeveloper**

### CODE REVIEW

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+* CSS Modules
+* LESS
+* SASS
+* ES7
+* MobX
+* Server Rendering
+
+..and a lot more out of the box!
+
+A lot of developers, including JavaScript newbies create *React* apps from scratch daily, so the *CRA* team at facebook built the *create-react-app* tool to make the process of creating such apps less tedious and error-prone.
```
Comment (brunokrebs) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: facebook ==> Facebook

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown [hunk 2]
```diff
@@ -0,0 +1,243 @@
+---
+layout: post
+title: "How to create a custom create-react-app script"
+description: "Create React App (CRA) is a very good tool for creating react apps from the CLI without build configuration. Learn how to create a custom script that works with create-react-app"
```
Comment (chenkie) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: Does Facebook actually use "CRA" as an acronym for it?

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown [hunk 3]
```diff
@@ -0,0 +1,243 @@
+---
+layout: post
+title: "How to create a custom create-react-app script"
```
Comment (chenkie) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: Let's give this a different title. How about: "Customizing create-react-app: How to Make Your Own create-react-app Template"

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+related:
+- introduction-to-progressive-apps-part-one
+- bootstrapping-a-react-project
+- secure-your-react-and-redux-app-with-jwt-authentication
+- adding-authentication-to-your-react-flux-app
+---
+
+---
+
+**TL;DR:** There are several tools available for developers to aid the building of various types of websites and applications. One of such is [**Create React App(CRA)**](https://github.com/facebookincubator/create-react-app), the CLI tool that helps JavaScript developers create **React** apps with no build configuration. As awesome as **CRA** is, developers still need a way of tweaking, adding special scripts and modules that doesn't come bundled with **CRA**. Today, I'll teach you how to create your own custom create-react-app scripts that you and your team harbor!
```
Comment (chenkie) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: `One of such is` => `One such tool is`

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+
+---
+
+Many developers already use [create-react-app](https://github.com/facebookincubator/create-react-app) to build their React applications, but like I mentioned earlier, developers are still screaming for more configurations! Some are interested in having support for:
```
Comment (chenkie) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: `configurations` => `configuration options`

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+
+Viola! You now have a fresh React app with full authentication powered by Auth0 ready for use. 
+
+[Sign up](javascript:signup\(\)) for a free account today and enjoy fast, seamless, and hassle-free authentication in your apps.
+
+## Conclusion
+
+Great programmers constantly sharpen their tools daily to increase productivity. *CRA* is a great tool for quickly building React Applications. In addition, having your own customized fork of `react-scripts` helps you and your team easily add all the configurations you need. You'll need to maintain your fork, and [make sure it is synced](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced) with the upstream to have all updates. [Backstroke](https://github.com/1egoman/backstroke) is a bot that can help you with this. 
+
+Have a very productive time hacking away!
```
Comment (chenkie) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: Looks great :) Thanks

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown (hunk 1 above)
Comment (unicodeveloper) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: Thanks

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown (hunk 3 above)
Comment (unicodeveloper) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: Oh that makes sense!

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown (hunk 2 above)
Comment (unicodeveloper) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: Yes, they do. Saw it been used  a lot in their pull requests

File: _posts/2017-01-11-how-to-configure-create-react-app.markdown (hunk 6 above)
Comment (unicodeveloper) [inline _posts/2017-01-11-how-to-configure-create-react-app.markdown]: Thanks

**Pull request merged**